### PR TITLE
add question-select component

### PR DESCRIPTION
**Pull request by neo-tahi**

### CODE REVIEW

**Comment by mikem** [comment]
Could you include a comment with instructions on how this component would be used? I realize that it's part of the work you're doing on the billing card, and that's where this component is being used.

Specifically, I'm curious how the selected data is passed.

**Comment by mikem** [comment]
Actually, it'd be great if you could include this component in the style guide. That would a) include it in the style guide, and b) provide one example in the code for how to use this component.

**Comment by 4thethrillofit** [comment]
## Preliminary documentation:

#### Basic usage
```hbs
{{question-select ident="namespace.identifier" 
  task=model 
  question="PlaceholderText" 
  class="any class you want to add to the select element" 
  selectionSelected='callbackWhenSelected'
  source=arrayOfObjWithIdAndText}}
```

`source` should look like
```javascript
var arrayOfObjWithIdAndText = [
  {id: 1, text: 'option one'},
  {id: 2, text: 'option two'}
]
```

The `selectionSelected` callback will pass back the data the user selected.

**Comment by mikem** [comment]
Thanks @4thethrillofit, that looks good.

**Comment by neo-tahi** [comment]
added this component to the styleguide

- @afomi

File: client/app/pods/components/question-select/component.coffee [hunk 1]
```diff
@@ -0,0 +1,8 @@
+`import QuestionComponent from 'tahi/pods/components/question/component'`
+
+QuestionSelectComponent = QuestionComponent.extend
+  selectedData: Em.computed 'model.answer', ->
+    @get('source').findBy 'id', parseInt(@get('model.answer'))
+
+
+`export default QuestionSelectComponent`
```
Comment (mikem) [inline client/app/pods/components/question-select/component.coffee]: This `QuestionSelect` looks different from what's in the [billing card branch currently](https://github.com/Tahi-project/tahi/blob/1318eae73599eb43927cc5f2312f8acd908dd90d/client/app/pods/components/question-select/component.coffee). Did this additional code get lost in the rebase of this branch?

**Comment by mikem** [comment]
Nice, glad to see this in the style guide. If there's no code missing from the component, this looks good to go. :+1:

**Comment by mikem** [comment]
Alright, this looks good to go. :+1:

**Pull request merged**